subsys/net/openthread/rpc/Kconfig

```diff
@@ -59,6 +59,18 @@ config OPENTHREAD_RPC_CLIENT_NUM_SENT_COAP_REQUESTS
 
 endmenu # "OpenThread over RPC client configuration"
 
+menu "OpenThread over RPC client configuration"
+	depends on OPENTHREAD_RPC_SERVER
+
+	config OPENTHREAD_RPC_MESSAGE_POOL
```

Comment:
Could you not indent options in `menu` to be consistent with the rest of NCS?

Reply:
fixed.